code/modules/pda/PDA.dm

```diff
@@ -464,9 +464,7 @@ var/global/list/obj/item/device/pda/PDAs = list()
 		pai.forceMove(T)
 	current_app = null
 	scanmode = null
-	for(var/A in programs)
-		qdel(A)
-	programs.Cut()
+	QDEL_LIST(programs)

```

Comment:
Oh welp, this one already merged... (I was looking at peoples code, to bone up my knowledge of what's going on)